crates/secrets-nats-kv/Dockerfile

```diff
@@ -23,4 +23,4 @@ WORKDIR /app
 COPY --from=builder /app/target/release/secrets-nats-kv .
 
 # Set the entrypoint
-ENTRYPOINT ["./secrets-nats-kv"]
\ No newline at end of file
+ENTRYPOINT ["./secrets-nats-kv", "run"]
```

Comment:
```suggestion
ENTRYPOINT ["./secrets-nats-kv"]

CMD ["run"]
```

Reply:
This allows folks to use the same docker image for CLI operations without having to override `entrypoint`